build.gradle

```diff
@@ -9,6 +12,11 @@ Base directory: $projectDir
 Running script ${relativePath(buildFile)}
 """
 
+if (file("vendor/protobuf").list().length == 0) {
+	throw new GradleException(
```

Comment:
If you have nothing against it, I can incorporate the codenarc plugin with the specification you told me for the style. This way we won't have the styling problem anymore as Gradle will catch them. What do you think?